src/app/client/src/app/plugins/profile/services/profile/profile.service.ts

```diff
@@ -125,4 +125,12 @@ export class ProfileService {
     };
     return this.learnerService.post(options);
   }
+
+  public fetchCertificates(request) {
```

Comment:
This method should go to CertRegService & call directly from component

Reply:
moved